applications/DEMApplication/custom_utilities/discrete_particle_configure.h

```diff
@@ -231,7 +226,7 @@ class DiscreteParticleConfigure
         SphericParticle* p_particle1 = static_cast<SphericParticle*>(&*rObj_1);
         SphericParticle* p_particle2 = static_cast<SphericParticle*>(&*rObj_2);
         const double radius_sum      = p_particle1->GetSearchRadius() + p_particle2->GetSearchRadius();
-        const bool intersect         = floatle((distance_2 - radius_sum * radius_sum),0);
+        const bool intersect         = floatle(distance_2, std::pow(radius_sum, 2));
```

Comment:
pow is slower with some compilers. We could leave radius_sum*radius_sum

Reply:
Really? I thought this was well taken care of by the compiler. I would prefer leaving it as pow; it is cleaner and will only get more optimized in time.

Reply:
I forgot to paste the comparison study: https://stackoverflow.com/questions/2940367/what-is-more-efficient-using-pow-to-square-or-just-multiply-it-with-itself

Reply:
My conclusion from the comparison is that indeed the difference is not really significant for 2. So I stick to my previous comments.